modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -151,6 +151,12 @@ public static void startup(Runnable runnable) {
      * {@link #startup(Runnable)} once.
      * </p>
      *
+     * <p>
+     * Memory consistency effects: Actions in a thread prior to submitting a
+     * runnable to this method <i>happen-before</i> actions performed by
+     * the runnable in the JavaFX Application Thread.
```

Comment:
I think it would be helpful to use code style for runnable on these two lines (since it refers to the `runnable` argument), like this: `{@code runnable}`

Reply:
Fixed. Whose responsibility is it to press the "Resolve" button?

Reply:
You can.